pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - "apps/*"
```

Comment:
apps?

Reply:
PR is WIP 
apps/ will be removed (used it to test nextjs-grafbase)

Reply:
@notrab I refactored your [POC](https://github.com/notrab/next-grafbase/), feel free to review here: [/packages/grafbase-nextjs-plugin](https://github.com/grafbase/grafbase/tree/gb-2713-setup-nextjs-grafbase-plugin/packages/grafbase-nextjs-plugin)